docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -21,64 +21,56 @@ tags:
+Each key is unique to a single identity and ties directly to their Role-Based Access Control (RBAC) settings, ensuring secure and appropriate access.
 
 The process is as follows:
 
 ```text
-API Key (authentication) → User (identity) → RBAC (authorization)
+API Key (authentication) → Identity (User or Service Account) → RBAC (authorization)
 ```
 
-These API Keys are integral for authenticating Temporal Cloud operator tools, including [Temporal Cloud CLI (tcld)](/cloud/tcld) and the [Temporal Cloud Operations API](/ops).
-
-Users have the autonomy to create, delete, and update access to their API Keys through the Cloud UI or tcld.
-
-API Keys are an opt-in feature that must be enabled by a Global Administrator for use.
-For broader oversight, Global Administrators can manage user's API Keys using the same interfaces.
-For instructions on API Key Management for your organization, see [global Administrator API Key Management](#global-administrator-api-key-management).
-
-## Get started {#get-started}
-
-To get started, ensure you're a Temporal Cloud user.
-If you don't have an account with Temporal, **[sign up here](https://pages.temporal.io/get-started-with-cloud)**.
-
-### Prerequisites
-
-- A Cloud user account
-- Access to the [Temporal Cloud UI](https://cloud.temporal.io/) or Temporal Cloud CLI ([tcld](https://docs.temporal.io/cloud/tcld/))
-- [Enable access to API Keys](#manage-api-keys)
+API Keys support 2 main use cases:
 
-## Use API Keys {#use-api-keys}
+- _**Cloud Operations Automation**_: API Keys can be used as the authentication mechanism for Temporal Cloud operational tools ([Temporal Cloud CLI (tcld)](/cloud/tcld), [the Cloud Ops API](/ops), [Terraform](/production-deployment/cloud/terraform-provider)) to manage your Temporal Cloud account, namespaces, certificates, and user identities.
+  - API Keys for use in Cloud Operations Tools are in Public Preview
+- _**Namespace Authentication**_: API Keys can be used as the Namespace authentication mechanism for executing and managing workflows from the SDK and Temporal CLI as an alternative to using mTLS-based authentication.
+  - API Keys for use in Namespace Authentication are in Pre Release
 
-API Keys can be used in the `tcld` or in the Cloud Ops API.
+API Keys are an opt-in feature that must be enabled by a Global Administrator for use. For instructions on API Key Management for your organization, see [Global Administrator API Key Management](#ga-manage-api-keys).
 
-**How can I use the API Key with tcld?**
+API Keys are supported for Users and Service Accounts.
 
-To use your API Key with tcld, use one of the following methods:
+- Users have the autonomy to create, delete, and update access to their API Keys through the Cloud UI or tcld. For broader oversight, Global Administrators can manage user's API Keys using the same interfaces.
```

Comment:
I would take away the period on these bullet points so it's consistent with the rest.

Reply:
"Autonomy" is a $5 word.

Users may create, delete, and update API key access using the Cloud Web UI or the `tcld` CLI tool.
Global Administrators have permission to manage user API keys using the same interfaces.

I'm using "Cloud Web UI" per previous tickets. Please check with team cloud to confirm.